submission/submitter/src/test/kotlin/ac/uk/ebi/biostd/submission/processors/AccNoPatternUtilTest.kt

```diff
@@ -21,21 +22,21 @@ import org.junit.jupiter.params.provider.CsvSource
 
 @ExtendWith(MockKExtension::class)
 class AccNoPatternUtilTest(
-    @MockK private val user: User,
-    @MockK private val context: PersistenceContext
+    @MockK private val mockUser: User,
+    @MockK private val mockPersistenceContext: PersistenceContext,
+    @MockK private val mockUserPrivilegesService: IUserPrivilegesService
```

Comment:
we have not followed this practice in all project name mocks starting with mocks.

Reply:
Mocks gonna mock